force-app/main/default/permissionsets/Monitor_Platform_Events.permissionset-meta.xml

```diff
@@ -4,6 +4,10 @@
         <apexClass>PlatformEventMonitoringController</apexClass>
         <enabled>true</enabled>
     </classAccesses>
+    <customPermissions>
+        <enabled>true</enabled>
+        <name>Monitor_Platform_Events</name>
+    </customPermissions>
```

Comment:
+1 for using a Custom Permission and not hardcoding the name of a PermissionSet or profile!